fdbclient/ServerKnobs.cpp

```diff
@@ -903,18 +903,18 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init ( CLUSTER_RECOVERY_EVENT_NAME_PREFIX,              "Master" );
 
 	// Encryption
-	init( ENABLE_ENCRYPTION,                                   false ); if ( randomize && BUGGIFY ) { ENABLE_ENCRYPTION = deterministicRandom()->coinflip(); }
+	init( ENABLE_ENCRYPTION,                                   false ); if ( randomize && BUGGIFY ) ENABLE_ENCRYPTION = true;
```

Comment:
Seems like this still needs an update?:)

Reply:
I replied to the comment above. The BUGGIFY macro has built-in random coin-flip, and we don't need another coin-flip on top of it to enable the knob by probability.

Reply:
`ENABLE_ENCRYPTION = true;` initial code was toggling the EncryptionState, with this update code will update the value to `true` avoiding `encryption=false` configuration testing once we update `Encryption` default to `true` ?

Reply:
When we update the default to `true`, we need to change it to `if (randomize && BUGGIFY) ENABLE_ENCRYPTION = false;`

Reply:
OR 

`init( ENABLE_ENCRYPTION,                                   false ); if ( randomize && BUGGIFY ) ENABLE_ENCRYPTION = !ENABLE_ENCRYPTION;` for simple toggle?